### ISSUE TITLE
Bug: Find not working when using deeper class inheritance 

### ISSUE THREAD

**Issue by Faberle** [jagi/meteor-astronomy#510]
Find doesn`t return the grand children of a class.

Example: 

``` javascript

const Parent = Class.create({
  name: 'Parent',
  fields: {
    parent: String
  }
});

const Child = Parent.inherit({
  name: 'Child',
  fields: {
    child: String
  }
});

const GrandChild = Child .inherit({
  name: 'GrandChild ',
  fields: {
    grandChild: String
  }
});

```

Parent.findOne({_id: [GrandChildId]}) //=> undefined
Child.findOne({_id: [GrandChildId]}) //=> found
GrandChild .findOne({_id: [GrandChildId]}) //=> found

https://github.com/jagi/meteor-astronomy/blob/v2/lib/modules/storage/class_static_methods/find.js#L49

Here we just take the next layer of children instead of recursively adding all children.

**Comment by Faberle** [comment]
I now added 

``` javascript
if (this.children) {
            let children = Array.from(this.children);
            _.each(children, child => children.push.apply(children, child.getChildren()));
            return children;
        }
```

(https://github.com/jagi/meteor-astronomy/blob/v2/lib/core/class.js#L138)
as a hotfix, but I don`t know if that causes bugs somewhere else.

**Comment by lukejagodzinski** [comment]
Hey, indeed there was a bug. Thanks! I've already fixed it but it's not published yet. However, you can clone the `v2` branch to local packages directory to have the newest version. I will publish it soon.

Right now you can decide on depth of children fetching.

``` js
Parent.find({}, {children: true}).fetch(); // All children
Parent.find({}, {children: false}).fetch(); // No children
Parent.find({}, {children: 1}).fetch(); // 1 level deep
Parent.find({}, {children: 2}).fetch(); // 2 level deep etc.
```

**Comment by lukejagodzinski** [comment]
Fixed and published in the 2.1.5
